### ISSUE TITLE
Incorrect merge of debuglog pull request from 2012?

### ISSUE THREAD

**Issue by Lerrissirrel** [munin-monitoring/munin-node-win32#88]
I feel like I must be missing something here.  After researching why I've been getting spurts of "Event ID 1" every few minutes from munin-node, I ran across the issue #4.  So I set the appropriate setting in munin-node.ini and it was still happening.  Figured I needed to start debugging the code, and along the way I realized that there are two places that call EventLog.Write in src/core/Services.cpp and I think the original pull request meant to modify CService::LogEvent() while the change actually landed in CService::LogError().

This leaves the (unconditional) LogEvents() from src/plugins/PerfCounterMuninNodePlugin.cpp continuing to log.   Now, it does appear that these non-conditional LogEvent() calls got added with commit 
098b0a5ab3a9c440e46914c7cafb85e8222079dd which came in Feb 2020, long after the original "fix" from issue #4.

Long story short, I don't know the intent of the original ini setting - disable Events only?  Errors only?  Both? Nor do I know the intent of the LogEvent()s in the plugin or what is considered the "correct" approach to errors/events in munin-node-win32.  So just alerting here for those more in the know!
